src/fsharp/service/SemanticClassification.fs

```diff
@@ -0,0 +1,183 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.SourceCodeServices
+
+[<RequireQualifiedAccess>]
+type SemanticClassificationType =
+    | ReferenceType
+    | ValueType
+    | UnionCase
+    | Function
+    | Property
+    | MutableVar
+    | Module
+    | Printf
+    | ComputationExpression
+    | IntrinsicFunction
+    | Enumeration
+    | Interface
+    | TypeArgument
+    | Operator
+    | Disposable
+
+[<AutoOpen>]
+module TcResolutionsExtensions =
+    open System.Diagnostics
```

Comment:
Generally we adopt the style of putting all `open`s at the top of the file, I think you can do that here

Reply:
Will do, if that's the general style. I'll try to keep that in mind, but I do prefer putting opens in modules like this; it doesn't matter that much though.